bundles/org.openhab.core.addon.xml/addon-1.0.0.xsd

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+           targetNamespace="https://openhab.org/schemas/addon/v1.0.0">
+
+    <xs:import namespace="https://openhab.org/schemas/config-description/v1.0.0"
+               schemaLocation="https://openhab.org/schemas/config-description-1.0.0.xsd"/>
+
+    <xs:element name="addon">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="type" type="addonType"/>
+                <xs:element name="name" type="xs:string"/>
+                <xs:element name="description" type="xs:string"/>
+                <xs:element name="author" type="xs:string" minOccurs="0">
+                    <xs:annotation>
+                        <xs:documentation>The organization maintaining the add-on (e.g. openHAB). Individual developer names should be avoided.</xs:documentation>
+                    </xs:annotation>
+                </xs:element>

```

Comment:
Sorry for commenting code already merged, but thought it would be easiest for proper context. Should it be mentioned that it's not needed to use "openHAB" as author here? At least it seems none of the add-ons in the openhab-addons repository have this, and the skeleton script doesn't create it.

Reply:
Maybe it would be the right time to drop this field now altogether - we more or less only kept it for backward-compatibility, but as we are breaking this now anyhow, I think we could remove it from the XSD.

Reply:
The marketplace add-on service will build the add-on with what Discourse returns as the author of the topic as the add-on author:

https://github.com/openhab/openhab-core/blob/8d64ecfd8db99844d79a776fef37cbe527c1f0c6/bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java#L334-L336

and the Karaf add-on service will put the constant "openHAB" in that field (and set it as "verified"):

https://github.com/openhab/openhab-core/blob/8f9dafd1d68a92248cd39414145175952d61cd46/bundles/org.openhab.core.karaf/src/main/java/org/openhab/core/karaf/internal/KarafAddonService.java#L133-L134

The author and the checkmark are displayed in the UI, notably in the information tables in each add-on page. For "verified" add-ons it's also displayed in the list.

Reply:
Thanks @ghys, this means that this field is definitely deprecated then.
@J-N-K Would you agree that this is a good moment to remove it from the XSD then?

Reply:
Yes